porcupine/plugins/filemanager.py

```diff
@@ -122,6 +157,31 @@ def rename(old_path: Path) -> None:
         tab.path = new_path / tab.path.relative_to(old_path)
 
 
+def paste(new_path: Path) -> None:
+    assert copy_path is not None
+    new_file_path = new_path / copy_path.name
+
+    if new_file_path.exists():
+        if new_file_path.parent == copy_path.parent:  # copying file in the same dir (sequential number copy)
```

Comment:
I don't like the sequential numbering, as a file named `foo (1).py` is unacceptable to me and I would have to rename it anyway. Can we just always ask for a new name?

Reply:
Perfect